### DISCUSSION TITLE
Welcome to DPC-Aggregati-Strutturali-ITC-NordOvest Discussions!

### DISCUSSION THREAD

**Discussion by pierluigicara** [pcm-dpc/DPC-Aggregati-Strutturali-ITC-NordOvest#1] [General]
## 👋 Benvenuti!
  Utilizzeremo le "discussion" come luogo per entrare in contatto tra tutti coloro che sono interessati al tema degli Aggregati Strutturali. 
Ci auguriamo che ciscuno:
   * ponga domande sull'argomento di cui è interessato.
   * condivida delle idee;
   * interagisca con altri membri della comunità;
   * accolga gli altri e abbia una mentalità aperta. 

  Voglio ricordare che questa è una modalità di interazione che stiamo sperimentando per la prima volta e dietro alla quale non c'è ancora una organizzazione strutturata e che ciascuno potrà contribuire a titolo volontario,

  Per cominciare mi presento e introduco anche quello che potrà essere il nucleo di persone che potranno insieme a me fare da punto di contatto per questo Repository. Mi riferisco in particolare ai referenti che ciascuna Regione/Provincia Autonoma ha indicato e che trovate elencati nella Sezione "Attori" del README. Insieme o - a seconda dei casi - suddivisi per ciascun territorio di propria competenza, potremo partecipare alle discussioni che si presenteranno dando ciascuno il proprio contributo per interagire con quella che potrà essere la comunità degli utilizzatori.
  io sono Pierluigi Cara (@pierluigicara per GitHub) e sono il referente per il DPC del Tavolo interregionale costituito per lavorare insieme alle regioni e province autonome sul tema della cartografia di supporto alle attività di rilievo del danno post evento. Tale cartografia è costituita dagli aggregati strutturali e dalle unità strutturali in essi contenute, alle quali durante i sopralluoghi per la verifica dell'agibilità viene attribuita una valutazione da parte delle squadre che effettuano i sopralluoghi medesimi. Siamo partiti dalla cartografia nazionale degli aggregati strutturali (che definiamo, appunto, di partenza) ma contiamo di proseguire il percorso valutando la possibilità di arrivare a definire per l'intero territorio nazionale anche una copertura delle unità strutturali. Naturalmente l'utilizzo di questi dataset può essere esteso rispetto al solo rilievo del danno post-sisma, nell'ambito della cosiddette attività di protezione civile, ma non escludiamo che possano venire dalla comunità degli utilizzatori anche ulteriori possibili scenari di utilizzo. Finchè questi resteranno nel "perimetro" delle attività di protezione civile potremmo trattarli, in caso contrario andranno trovate delle possibili alternative.
  Per partecipare, dunque, vi invito a registrare un vostro account su GitHub (se non lo avete già fatto).
  Per ulteriori informazioni potete consultare la documentazione che GitHub rende disponibile [link to the documentation](https://docs.github.com/en/discussions/quickstart).

**Comment by skampus** [comment]
Non apro nuovo thread per non creare subito "confusione".
Molto utile questa sezione di GitHub che spero possa diventare un po' il riferimento per tutto ciò che riguarda Aggregati Strutturali e tutto ciò che ruota attorno (aggiornamenti, problemi interpretativi, specifiche, eccetera eccetera...).

Complimenti per l'apparato documentale che accompagna i dataset!

s.

ps: sono stefano campus, funzionario del Settore Sistema Informativo Territoriale e Ambientale di Regione Piemonte che cura il database topografico regionale (BDTRE) e che è fonte degli aggregati strutturali del Piemonte.
